src/Frontend/Core/Engine/Page.php

```diff
@@ -117,8 +118,17 @@ public function load(): void
         // create header instance
         $this->header = new Header($this->getKernel());
 
+        try {
+            $this->handlePage(Navigation::getPageId(implode('/', $this->url->getPages())));
+        } catch (NotFoundHttpException $notFoundHttpException) {
+            $this->handlePage(Response::HTTP_NOT_FOUND);
```

Comment:
I assume the constant `Response::HTTP_NOT_FOUND` is 404? What if they change the value of the constant? I think the purpose of using class-constants is to be able to change the internal value without breaking the public API

Reply:
fixed in cf7eaae29